.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [18.x]
```

Comment:
Let's keep Node 16 here, as discussed in the PR.

Reply:
Test matrix updated `[16.x, 18.x]`